src/components/common/time/TimeField.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import dayjs from "dayjs";
+import { FormControl } from "@mui/material";
+import { TimePicker } from "@mui/x-date-pickers";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import "twin.macro";
+import "styled-components/macro";
+import { InputLabel } from "@madie/madie-design-system/dist/react/";
+import { kebabCase } from "lodash";
+
+export const timeFieldStyle = {
+  borderRadius: "3px",
+  height: 40,
+  width: 134,
+  border: "1px solid #DDDDDD",
+  "& .MuiOutlinedInput-notchedOutline": {
+    borderRadius: "3px",
+    "& legend": {
+      width: 0,
+    },
+  },
+  "& .MuiOutlinedInput-root": {
+    "&&": {
+      borderRadius: "3px",
+    },
+  },
+  "& .MuiInputBase-input": {
+    color: "#333",
+    fontFamily: "Rubik",
+    fontSize: 14,
+    borderRadius: "3px",
+    padding: "9px 5px 9px 5px",
+    "&::placeholder": {
+      opacity: 1,
+      color: "#717171",
+      fontFamily: "Rubik",
+      fontSize: 14,
+      padding: "9px 5px 9px 5px",
+    },
+  },
+};
+interface TimeFieldProps {
+  label: string;
+  time: string;
+  handleTimeChange: Function;
+  canEdit: boolean;
+  required: boolean;
+}
+
+const TimeField = ({
```

Comment:
Since this is a primitive component, and like other components such as DateTimeField and DateField are implemented in MadieDesignSystems, Does it make sense to move this to design systems ?

Reply:
@RohitKandimalla Yeah it does make sense to move it to design systems. Thanks a lot Rohit!